agent/run.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+# The script is used for manual testing of the agent.
+java -jar target/remoting-kafka-agent-1.0-SNAPSHOT-jar-with-dependencies.jar -name test -master http://localhost:9090 -kafkaURL 127.0.0.1:9092
```

Comment:
Eventually it might be useful to pull some of these values out as properties or variables in the script. Or even accept them as command-line arguments.

This form is definitely acceptable in this PR for now.